data/domain/rift.tsx

```diff
@@ -58,11 +61,10 @@ export class InfiniteStarsBonus extends RiftBonus {
 }
 
 // Use a typeguard to prove a skill index is a valid rift mastery skill
-const isRiftMasterySkill = (x: SkillsIndex): x is RiftMasterySkills => 
- ![SkillsIndex.Farming, SkillsIndex.Summoning, SkillsIndex.Sneaking].includes(x)
+const isRiftMasterySkill = (x: SkillsIndex): x is RiftMasterySkills => true;
 
-// Rift doesn't care about the new w6 skills for now, so exclude them from our index.
-type RiftMasterySkills = Exclude<SkillsIndex, SkillsIndex.Farming | SkillsIndex.Summoning | SkillsIndex.Sneaking>
+// Leaving this in case in need of hiding some other skills
+type RiftMasterySkills = SkillsIndex
```

Comment:
oh I didn't even know they were added to the rift lol.

Reply:
I noticed that on Toolbox when I glanced at it for Farming, then checked in-game and saw that they're indeed in the game x)